testing/jormungandr-testing-utils/src/testing/node/rest/raw.rs

```diff
@@ -187,6 +187,7 @@ impl RawRest {
     }
 
     pub fn send_fragment(&self, fragment: Fragment) -> Result<Response, reqwest::Error> {
+        #[allow(unused_imports)]
         use chain_core::property::Fragment as _;
```

Comment:
Why is this needed?

Reply:
Looks like for nothing. They were some leftovers and I just automatically added the macro flag. Gonna be removed